flow/__main__.py

```diff
@@ -46,6 +49,43 @@ def main_init(args):
         )
 
 
+def main_template(args):
+    """Print out general information about template subcommand."""
+    print("flow template does nothing. Use of the available subcommands [create,].")
```

Comment:
This feels clunky. I think there's a more argparse-friendly way to show help messages? You might need to do something with the subparser. I can look into this if you don't see a solution.